### PR TITLE
Pl 1497 bug fix

### PR DESCRIPTION
**Pull request by sumanmukherjee03**

### CODE REVIEW

File: integrations/ci/circle.go [hunk 1]
```diff
@@ -110,8 +125,9 @@ func (c *Circle) GetCompletedBuild(m *core.Manifest) (*circleci.Build, error) {
 		return build, err
 	}
 	if !exists {
-		log.Printf("CircleCI not configured. Skipping check...")
-		return build, nil
+		errMsg := "CircleCI not configured. Skipping check..."
```
Comment (philwhln) [inline integrations/ci/circle.go]: `Skipping check...` probably not needed.

File: integrations/ci/circle.go [hunk 2]
```diff
@@ -110,8 +125,9 @@ func (c *Circle) GetCompletedBuild(m *core.Manifest) (*circleci.Build, error) {
...
+		log.Printf(errMsg)
+		return build, NewCircleCINotConfiguredError(errMsg)
```
Comment (philwhln) [inline integrations/ci/circle.go]: Could move this to `configurationExist`

Comment (sumanmukherjee03) [reply integrations/ci/circle.go]: Done 👍

File: integrations/ci/circle.go (hunk 1 above)
Comment (sumanmukherjee03) [reply integrations/ci/circle.go]: Kept this as part of the logging but not the error message. It is helpful to see that the check is being skipped in the jenkins logs.

**Pull request merged**